### ISSUE TITLE
Update Turkish README.md

### ISSUE THREAD

**Issue by Lendemor** [reflex-dev/reflex#3320]
Update the `docs/tr/README.md` to match the latest version of the `README.md` in #3313

**Comment by hasansezertasan** [comment]
> Update the `docs/tr/README.md` to match the latest version of the main `README.md`

If you are going to keep opening "Update {{ language }} README.md" whenever the main `README.md` gets updated, I believe you should put the issue number in the title and description.

Something like:

```markdown
Update the `docs/tr/README.md` to match the latest version (#3234) of the main `README.md`.
```

**Comment by Lendemor** [comment]
> > Update the `docs/tr/README.md` to match the latest version of the main `README.md`
> 
> If you are going to keep opening "Update {{ language }} README.md" whenever the main `README.md` gets updated, I believe you should put the issue number in the title and description.
> 
> Something like:
> 
> ```
> Update the `docs/tr/README.md` to match the latest version (#3234) of the main `README.md`.
> ```

Good call, I'll had the commit hash of the specific version to update to in the messages once #3234 is merged.

**Comment by hasansezertasan** [comment]
> Good call, I'll had the commit hash of the specific version to update to in the messages once #3313 is merged.

Great, how often does the main `README.md` get updated?

**Comment by Lendemor** [comment]
Not that often, it is pretty stable now.

We can just expect a few code changes in the demo snippet when we release new 0.X version, but shouldn't change much aside from that.

**Comment by hasansezertasan** [comment]
> Not that often, it is pretty stable now.

🚀 🚀 🚀 

> We can just expect a few code changes in the demo snippet when we release new 0.X version, but shouldn't change much aside from that.

I see.

> Main reason for opening the issues was that a lot of languages were outdated compared to the main ones.

I asked because I was gonna recommend creating an automation script for the process.
